### PR TITLE
🔨 Refactor, 🧠 Overmind, Hacktoberfest - /app/pages/common/Modals/LiveSessionEnded/index.js: convert to tsx and uses overmind

### PR DESCRIPTION
**Pull request by ftonato**
Co-authored-by: Eliamar Tani <eliamartani@gmail.com>

## What kind of change does this PR introduce?

Refactors code as a part of hacktoberfest mentioned in #2621.
@Saeris @christianalfoni

## What is the current behavior?

/app/pages/common/Modals/LiveSessionEnded/index.js was using inject from app/componentConnectors and written in JS.

## What is the new behavior?

uses useOvermind hook from app/overmind and converted to tsx.

## Checklist
* [ ]  Documentation
* [ ]  Testing
* [x]  Ready to be merged
* [ ]  Added myself to contributors table

### CODE REVIEW

**Comment by ftonato** [comment]
@lbogdan should I solve the conflict or you'll fix it?

**Comment by lbogdan** [comment]
@ftonato You should probably merge this PR with latest master.

**Comment by ftonato** [comment]
@lbogdan I have made this change, please could you verify again?

File: packages/app/src/app/pages/common/Modals/PRModal/index.tsx [hunk 1]
```diff
@@ -1,12 +1,16 @@
-import React from 'react';
-import { inject, observer } from 'app/componentConnectors';
 import { GitProgress } from 'app/components/GitProgress';
+import { useOvermind } from 'app/overmind';
+import React, { FunctionComponent } from 'react';
+
+export const PRModal: FunctionComponent = () => {
```
Comment (SaraVieira) [inline packages/app/src/app/pages/common/Modals/PRModal/index.tsx]: There are files importing this that will fail

File: packages/app/src/app/pages/common/Modals/LiveSessionEnded/index.tsx [hunk 2]
```diff
@@ -5,7 +5,7 @@ import React, { FunctionComponent } from 'react';
 import { Explanation, Heading } from '../elements';
 import { Container } from './elements';
 
-const LiveModeEnded: FunctionComponent = () => {
+export const LiveModeEnded: FunctionComponent = () => {
```
Comment (SaraVieira) [inline packages/app/src/app/pages/common/Modals/LiveSessionEnded/index.tsx]: Same

Comment (ftonato) [reply packages/app/src/app/pages/common/Modals/LiveSessionEnded/index.tsx]: Done ;)

File: packages/app/src/app/pages/common/Modals/PRModal/index.tsx (hunk 1 above)
Comment (ftonato) [reply packages/app/src/app/pages/common/Modals/PRModal/index.tsx]: Done @SaraVieira o/

**Comment by lbogdan** [comment]
Build for latest commit https://github.com/codesandbox/codesandbox-client/pull/2810/commits/9ed9e2725ffa2ef8937902e6139b7c772fd7c6d9 is at https://pr2810.build.csb.dev/s/new.

**Comment by ftonato** [comment]
Hello @CompuIves, can we merge now?

**Comment by ftonato** [comment]
Hello @CompuIves, @Saeris, @christianalfoni, can we merge now?

**Comment by MichaelDeBoey** [comment]
Duplicate of #2807

**Pull request closed without merging**